test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/Dnx.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Dnx.Runtime;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class Dnx
+    {
+        private readonly string _sdkPath;
+
+        public Dnx(string sdkPath)
+        {
+            _sdkPath = sdkPath;
+        }
+
+        public ExecResult Execute(string commandLine, bool dnxTraceOn = true)
+        {
+            string command;
+            if (RuntimeEnvironmentHelper.IsWindows)
+            {
+                command = "cmd";
+                commandLine = $"/C {Path.Combine(_sdkPath, "bin", "dnx.exe")} {commandLine}";
+            }
+            else
+            {
+                command = "bash";
```

Comment:
This will also break on CoreCLR on linux/osx where we actually have a native executable.